### PR TITLE
Avail Table Tennis (ping pong)

### PR DESCRIPTION
**Pull request by fabiooshiro**

### CODE REVIEW

File: internal/sequencers/avail/avail_listener.go [hunk 1]
```diff
@@ -32,24 +31,29 @@ const (
...
+	L1CurrentBlock  uint64
 }
 
 type PaioDecoder interface {
 	DecodePaioBatch(bytes string) (string, error)
 }
 
-func NewAvailListener(fromBlock uint64, repository *cRepos.InputRepository, w *inputter.InputterWorker) supervisor.Worker {
+func NewAvailListener(availFromBlock uint64, repository *cRepos.InputRepository, w *inputter.InputterWorker, fromBlock uint64) supervisor.Worker {
+	paioDecoder := ZzzHuiDecoder{}
```
Comment (ZzzzHui) [inline internal/sequencers/avail/avail_listener.go:50]: haha 😂

Comment (sandhilt) [reply internal/sequencers/avail/avail_listener.go:50]: @fabiooshiro He likes making tributes, like 'brunodo,' where 'bru' is part of my name.

**Review by sandhilt — APPROVED** [review]
LGTM 👍🏽

File: internal/sequencers/avail/avail_listener.go [hunk 2]
```diff
@@ -32,24 +31,29 @@ const (
 	DELAY                   = 500
 	ONE_SECOND_IN_MS        = 1000
 	FIVE_MINUTES            = 300
+	// FIVE_MINUTES = 30
```
Comment (sandhilt) [inline internal/sequencers/avail/avail_listener.go:34]:
```suggestion
```

File: internal/sequencers/avail/avail_listener.go (hunk 1 above)
Comment (ZzzzHui) [reply internal/sequencers/avail/avail_listener.go:50]: Love that one too. Brunodo, its perfect LOL

**Pull request merged**